packages/paste-core/components/anchor/src/index.tsx

```diff
@@ -22,21 +24,30 @@ export const secureExternalLink = (href: string): {} | undefined => {
   };
 };
 
-const Anchor = React.forwardRef<HTMLAnchorElement, AnchorProps>(({variant = 'default', ...props}, ref) => {
-  const AnchorComponent = AnchorVariants[variant];
-
-  return (
-    <AnchorComponent
-      href={props.href}
-      ref={ref}
-      variant={variant}
-      {...secureExternalLink(props.href)}
-      {...safelySpreadTextProps(props)}
-    >
-      {props.children}
-    </AnchorComponent>
-  );
-});
+const Anchor = React.forwardRef<HTMLAnchorElement, AnchorProps>(
+  ({variant = 'default', showExternal, ...props}, ref) => {
```

Comment:
Instead of the `showExternal` prop, could we somehow have the `secureExternalLink` function return the `Box` and `LinkExternalIcon`? Maybe thats a too heavy handed though since it's a visual change.

Reply:
So I didn't want it to be automatic based on those folks who are already manually showing the icon. If they update I don't want them having 2 icons, as that's a breaking change.

I also want the option to not show it, for example if I wrap a block level object like an image that links externally, it might not make sense to show the icon there

Reply:
Right. I wasn't thinking about the possible breaking change. It makes sense to have the prop.